enhancements/machine-api/nutanix-multi-nics.md

```diff
@@ -0,0 +1,128 @@
+---
+title: nutanix-multi-nics
+authors:
+reviewers:
+approvers:
+api-approvers:
```

Comment:
These need to be filled out, I can be assigned as review and api-approver. Probably this needs some installer input so @patrickdillon probably wants to be the approver here, unless he wants to delegate to another

Reply:
Yes I can be a reviewer. Also @rvanderp3 or someone from splat